wavefront_pyformance/wavefront_reporter.py

```diff
@@ -137,48 +89,25 @@ class WavefrontDirectReporter(WavefrontReporter):
     def __init__(self, server, token, source='wavefront-pyformance',
                  registry=None, reporting_interval=10, clock=None,
                  prefix='direct.', tags=None):
-        """Run parent __init__ and do direct reporter specific setup."""
-        super(WavefrontDirectReporter, self).__init__(
-            source=source,
-            registry=registry,
-            reporting_interval=reporting_interval,
-            clock=clock,
-            prefix=prefix,
-            tags=tags)
         self.server = self._validate_url(server)
         self.token = token
         self.batch_size = 10000
-        self.headers = {'Content-Type': 'text/plain',
-                        'Authorization': 'Bearer ' + token}
-        self.params = {'f': 'graphite_v2'}
+        super(WavefrontDirectReporter, self).__init__(
```

Comment:
Call `super()` first and then add extra attributes, and bring back the docstring